kconfigs/xen.conf

```diff
@@ -0,0 +1,30 @@
+CONFIG_EXPERT=y
+CONFIG_TEE=y
+CONFIG_OPTEE=y
+
+# CONFIG_ACPI is not set
```

Comment:
Do we need all these commented out config variables? We don't keep them for the kernel or U-Boot.

Reply:
These are needed here as when we enable CONFIG_EXPERT option (required for TEE and OPTEE) , 
these configs get added and default values of these options is not assumed and build stops to ask user for these.
I didn't explore it much but at that time adding these options here stopped build from halting to ask for these values. I will explore xen a little more to see if we can do it in a better way.

Reply:
I checked in xen again , few of the config options like ACPI and others mentioned in the conf file don't have default values set and are enabled only with EXPERT option. So, we will have to add them in .conf file else build will stop to ask for these options.

Reply:
That's fine, my point was to have just the ones needed or this config might only work with the the current Xen version, which would make upgrades harder than necessary.

Reply:
If I keep only the EXPERT, TEE and OPTEE options, xen build will stop in middle to ask for these config values  which user will have to  select. Would that work with the CI ?

Reply:
No, it must be fully automated. Worst case you need to keep this, but then you'll need to leave detailed instructions on how to generate that this file so it can be re-generated when upgrading Xen. Surely there must be a nice way of doing this.

Reply:
I double checked win xen repo. Right now TEE Mediator support in XEN is in Tech Preview stage. Hence it can be enabled only if CONFIG_EXPERT=y is selected i.e expert mode. With expert mode, this issue will remain. As more features are added in subsequent xen releases for expert mode, user will be prompted for the options which don't have default values in xen kconfig files and either these options have to be manually selected or corresponding option has to be added in the build/kconfig/xen.conf for now.